client/src/components/Overview/subcomponents/DropDown.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+// import css from '../styles/sizeSelector.module.css';
+
+class Dropdown extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showMenu: false
+    };
+
+    this.toggleMenu = this.toggleMenu.bind(this);
+    this.closeMenu = this.closeMenu.bind(this);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.closeMemu);
+  }
+
+  closeMenu() {
+    this.setState({ showMenu: false }, () => {
+      document.removeEventListener('click', this.closeMemu);
+    });
+  }
+
+  toggleMenu(e) {
+    console.log('ss')
+    e.stopPropagation();
+    e.nativeEvent.stopImmediatePropagation();
```

Comment:
I'm seeing this error on the browser when clicking on the button to toggle photos, sizes or quantity
![image](https://user-images.githubusercontent.com/17188626/147692875-a0a2d33e-376c-4014-94a3-65a345274acd.png)